ramp-frontend/ramp_frontend/tests/test_auth.py

```diff
@@ -451,6 +450,13 @@ def test_reset_password(client_session):
                 reset_password_link.find("/reset") :
             ]
 
+    rv = client.post("/reset_password", data={"email": user.email})
```

Comment:
This would have sent an email and we would have 2 emails in the outbox instead of one. I assume that the test was passing because we were blocking the execution in some way :)